adapters/Vungle/VungleAdapter/VungleRouterConsent.m

```diff
@@ -13,23 +13,24 @@
 // limitations under the License.
 
 #import "VungleRouterConsent.h"
-#import <VungleSDK/VungleSDK.h>
-
-static VungleConsentStatus currentConsentStatus = 0;
+#import <VungleAdsSDK/VungleAdsSDK.h>
 
 @implementation VungleRouterConsent
-+ (BOOL)updateConsentStatus:(VungleConsentStatus)consentStatus {
-  VungleSDK *sdk = VungleSDK.sharedSDK;
-  if (consentStatus > 0) {
-    currentConsentStatus = consentStatus;
-    [sdk updateConsentStatus:consentStatus consentMessageVersion:@""];
-    return YES;
-  }
-  return NO;
+
++ (void)updateGDPRStatus:(BOOL)optIn {
+  [VunglePrivacySettings setGDPRStatus:optIn];
+}
+
++ (void)updateCCPAStatus:(BOOL)optIn {
+  [VunglePrivacySettings setCCPAStatus:optIn];
+}
+
++ (void)updateCOPPAStatus:(BOOL)optIn {
+  [VunglePrivacySettings setCOPPAStatus:optIn];
 }
 
-+ (VungleConsentStatus)getConsentStatus {
-  return currentConsentStatus;
++ (void)updateIDFVStatus:(BOOL)optIn {
+  [VunglePrivacySettings setPublishIdfv:optIn];
```

Comment:
If these methods are static and can be called at any time (before or after initialization), I think publishers can just call them directly instead of needing a wrapper API?
If so, then we can just remove this class entirely.

Reply:
The methods in the Vungle SDK are static. We have a publisher request to be able to set the privacy values through the adapter code rather than communicating directly with the SDK API, which was why this class was created before to set the GDPR status. The RouterConfiguration class was removed to condense the IDFV setting to this class too.